.circleci/config.yml

```diff
@@ -282,6 +314,7 @@ jobs:
     executor: python38
     steps:
       - run_test:
+          store_coverage: false
```

Comment:
Probably a sign that we shouldn't be using `run_test` for these, I'll follow up with a PR.

Reply:
👍🏻 yeah agreed, since we are working some specific logic in here